pkg/Makefile

```diff
@@ -65,13 +66,15 @@ tar: prep
 		--license $(LICENSE) \
 		--url $(URL) \
 		--vendor $(VENDOR) \
-		--package $(TARGET_DIR)/asconfig-$(VERSION).tar
+		--package $(TARGET_DIR)/asconfig-$(VERSION)-$(ARCH).tar
 
 .PHONY: prep
 prep:
 	install -d $(TARGET_DIR)
 	install -d $(BUILD_DIR)/opt/aerospike/bin
 	install -pm 755 $(TOP_DIR)/asconfig $(BUILD_DIR)/opt/aerospike/bin
+	install -d $(BUILD_DIR)/usr/bin
```

Comment:
Weird that you need to create this empty directory but I am guessing it just looks at the file structure and replicates it

Reply:
yeah, fpm structures everything based on the directory, and I need that their so I can symlink against it.